### ISSUE TITLE
Nan Loss 

### ISSUE THREAD

**Issue by xxiMiaxx** [NNDam/adaface-partialfc#4]
Hello @NNDam 

Thank you so much for the great implementation,

I'm using your repo to train webface42m with pfc+adaface, with the config files from insightface, i've noticed two issues:

-  the loss keeps converging to Nan, i'm currently experimenting with lr 0.1 to see what happens
- one of the models currently training at epoch 12 and is fixed with a learning rate of 0.4, lr is not decreasing 

let me know if you faced the same issue.

Thank you!

**Comment by NNDam** [comment]
Same problem. Looklike cosface-pfc + augment still a best choice for me :)

**Comment by xxiMiaxx** [comment]
Hi, just want you to know that we fixed it a while ago, I believe it's something with lr_scheduler not being called properly.

**Comment by NNDam** [comment]
@xxiMiaxx Glad to hear that. It would be very grateful if you can share something about your experiments

**Comment by deep-practice** [comment]
Is the model trained  with pfc+adaface better than pfc+cosface for webface42m dataset? @xxiMiaxx
